File: src/test/java/ukalus/persistence/ThrowException.java
/*
 * Created on Dec 28, 2004
 *
 */
package ukalus.persistence;

import java.util.function.Function;

/**
 * @author pmgremo
 * 
 */
public class ThrowException implements Function<Reference, Object> {

  private static final long serialVersionUID = 3690754012043620921L;

  public Object apply(Reference object) {
    throw new RuntimeException();
  }

}

File: src/test/kotlin/ukalus/persistence/ThrowException.kt
/*
 * Created on Dec 28, 2004
 *
 */
package ukalus.persistence

import java.io.Serializable
import java.util.function.Function

/**
 * @author pmgremo
 */
class ThrowException<T> : Function<Reference<T?>, Any?>, Serializable {
    override fun apply(`object`: Reference<T?>): Any? {
        throw RuntimeException()
    }
}